dlp/snippets/inspect/inspect_bigquery.go

```diff
@@ -85,28 +85,28 @@ func inspectBigquery(w io.Writer, projectID string, infoTypeNames []string, cust
 	// Create a PubSub Client used to listen for when the inspect job finishes.
 	pubsubClient, err := pubsub.NewClient(ctx, projectID)
 	if err != nil {
-		return fmt.Errorf("pubsub.NewClient: %w", err)
+		return fmt.Errorf("pubsub.NewClient: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
Fixed it.